src/data/markdown/docs/02 javascript api/11 k6-execution/1-Execution.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Execution
+excerpt: 'Get execution information'
+---
+
+k6 v0.34.0 introduced the capability for getting information about the current test execution state, from inside of the test's script.
+
+It can be used by importing the `k6/execution` module.
+
+<div class="code-group" data-props='{"labels": [], "lineNumbers": [true]}'>
+
+```js
```

Comment:
Apparently Gatsby doesn't understand `js`. And the other code blocks should have it as well.

```suggestion
```javascript
```